README.md

```diff
@@ -34,15 +34,20 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+Responsive is fluid and adapts to the sscreen size of any device. Responsive uses CSS media queries to change styles based on the target device such as display type, width, height, etc., and only one of these is necessary for the site to adapt to different screens.
+
 2. Describe what it means to be mobile first vs desktop first.
+   Mobile means you build the site according to mboile deivces first then change it around for desktop.
+   Desktop first means vice-versa.
 
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
+   Rem either increases it or decreases the size. This makes 1 rem = 10px.
 
-4. How would you describe preprocessing to someone new to CSS?
+4. How would you describe preprocessing to someone new to CSS? Its almost close to having HTML javascript and CSS in one at a very beginner level. You need to be well organised for the codes to look organised to you.
```

Comment:
Nice attempt. This could be better explained in [this article](https://htmlmag.com/article/an-introduction-to-css-preprocessors-sass-less-stylus)